tsdb/operations/default.json

```
@@ -22,4 +22,52 @@
         },
         "size": 1000
       }
-    }
\ No newline at end of file
+    },
+    {
+      "name": "date-histo-entire-range-1m",
+      "operation-type": "search",
+      "index": "tsdb-1m",
+      "body": {
+        "size": 0,
+        "aggs": {
+          "date": {
+            "date_histogram": {
+              "field": "@timestamp",
+              "fixed_interval": "1m"
+            }
+          }
+        }
+      }
+    },
+    {
+      "name": "date-histo-entire-range-1h",
+      "operation-type": "search",
+      "index": "tsdb-1h",
+      "body": {
+        "size": 0,
+        "aggs": {
+          "date": {
+            "date_histogram": {
+              "field": "@timestamp",
+              "fixed_interval": "1h"
+            }
+          }
+        }
+      }
+    },
+    {
+      "name": "date-histo-entire-range-1d",
+      "operation-type": "search",
+      "index": "tsdb-1h",
```

did you intend 
```suggestion
      "index": "tsdb-1d",
```
?